### ISSUE TITLE
Acronym markup

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#114]
Consider adding mark-up for acronyms.  Possibly generate html tags like:

```
#!html

   <acronym title="Last-in First-out">LIFO</acronym>
```

and having the title display when the read does a hover.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/114
- Originally reported by: [rhettinger](https://bitbucket.org/rhettinger)
- Originally created at: 2009-03-04T00:14:38.883

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2009-03-05 08:49:26+00:00_

I implemented an :abbr: role in <<cset 51265dfe3f7aca6f4253db7216eaad95814d1729>>.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2010-08-25 12:19:55+00:00_

Removing milestone: 0.6 (automated comment)
